README.md

```diff
@@ -1,12 +1,10 @@
 # Business Site
 
```

Comment:
Blue Level Requirements
🔵 Blue Level (Minimum Requirements)
• Style your page -ok!
• Site should have a header with responsive image/video -ok!
• Site should have a signup form with at least three different input types -ok!
• Site should have an FAQ accordion, built with JavaScript -ok!

Great Job Rebecca!! Let me know if you have any questions or feedback on my code review.